### PR TITLE
fix(ci) Clean up Windows artifacts, installer, LLVM etc.

### PR DESCRIPTION
**Pull request by Hywan**
# Description

1.  7ff16b4, First, we need to use our own LLVM built for Windows. This has been commited by error, see https://github.com/wasmerio/wasmer/pull/2008#discussion_r555588703.
2. 3946f3c, Second, we copy the `ATTRIBUTIONS` and `LICENSE` files twice. That's useless :-).
3. efff2c8 ,  Third, as discussed with @MarkMcCaskey, we want to rename `wasmer-windows.exe` to `wasmer-windows-installer.exe` for the sake of clarity.
4. 1a03e88, Fourth, let's create a `wasmer.tar.gz` archive for Windows.
5. 67e8679, Fiveth, let's upload the `wasmer.tar.gz` archive for Windows.
6. cf0c56c, Use `wasmer.zip` on Windows because it's difficult to untar+ungzip on this OS (as discused with @MarkMcCaskey).

# Review

- [ ] ~Add a short description of the the change to the CHANGELOG.md file~ not necessary

### CODE REVIEW

**Review by MarkMcCaskey — APPROVED** [review]
Looks good to me, this seems like the kind of thing @syrusakbary might care about too though

**Comment by Hywan** [comment]
bors try

**Comment by Hywan** [comment]
bors try

**Comment by Hywan** [comment]
bors try-

**Comment by Hywan** [comment]
bors try

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -356,18 +352,28 @@ jobs:
...
+      - name: Upload Release Asset Windows Installer
         uses: actions/upload-release-asset@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }} # This pulls from the CREATE RELEASE step above, referencing it's ID to get its outputs object, which include a `upload_url`. See this blog post for more info: https://jasonet.co/posts/new-features-of-github-actions/#passing-data-to-future-steps 
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
           asset_path: artifacts/wasmer-windows-amd64/WasmerInstaller.exe
-          asset_name: wasmer-windows.exe
+          asset_name: wasmer-windows-installer.exe
```
Comment (jubianchi) [inline .github/workflows/main.yaml:363]: This may break `wasmer-install`: https://github.com/wasmerio/wasmer-install/blob/master/install.ps1#L42

**Review by jubianchi — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yaml (hunk 1 above)
Comment (syrusakbary) [reply .github/workflows/main.yaml:363]: Yup we will need to update it prior the next release

**Review by syrusakbary — APPROVED** [review]

File: .github/workflows/main.yaml (hunk 1 above)
Comment (Hywan) [reply .github/workflows/main.yaml:363]: Agree. Even if we update `install.ps1`, we must keep backward compatibility. I'll revert that commit.

**Comment by Hywan** [comment]
bors try

**Review by jubianchi — APPROVED** [review]

**Pull request closed without merging**